docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+* - All projects requiring a TRE must know and accept the financial cost of doing so
+  - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
+  - Mandatory
+* - You should be able to track costs associated with each TRE project
+  - This includes knowing which costs are associated with which project, and having an appropriate charging mechanism in place in line with your organisational policy
+  - Recommended
+* - You should have a process in place to ensure your TRE provision remains sustainable
+  - This could include having a cost recovery process in place, or setting up a long-term funding mechanism to support projects with TREs. At any given time, you should have funds free to cover all potential foreseen TRE provision for at least 12 months
+  - Recommended
+* - You should minimise the cost of your TRE infrastructure wherever possible
+  - You should have regular reviews of your TRE provision and actively work to bring down costs, streamline provision, and optimise support
+  - Optional
+```
+
+## Procurement
+
+_What the organisation does to ensure the effective sourcing, purchasing and supply of the goods and services that enable the organisation to operate._
+
+```{list-table}
+:header-rows: 1
+:name: tab-procurement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## IT Service management
+
+_The implementation and management of quality IT services that meet the needs of the organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-it-service-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a team in place to support projects working with TREs
+  - This may be part of your organisation's IT support team, or separate. Responsibility should be clear and stakeholders should easily be able to access support appropriate to their needs
+  - Mandatory
+```
+
+## Relationship management
+
+_All activities aimed at ensuring a continuous level of engagement is maintained between the organisation and its customers, stakeholders & other interested parties._
+
+### Stakeholder relationships
+
+```{list-table}
+:header-rows: 1
+:name: tab-relationship-stakeholder
+
+* - Statement
+  - Guidance
+  - Importance
+* - You should have a clear process in place for stakeholders to feedback on your TRE infrastructure
+  - This may include a GitHub repository where people can open issues and discussions, communication streams like Slack or email, or forms stakeholders can fill in
+  - Recommended
+```
+
+### Patient and Public Engagement
+
+_What the organisation does to ensure it establishes and maintains mutually beneficial relationships between the organisation and the public in order to maintain trust._
+
+```{list-table}
+:header-rows: 1
+:name: tab-patient-public-engagement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
```

Comment:
```suggestion
* - The TRE organisation should develop a plan for engagement with the public.
  - This may involve consulting members of the public, holding public events or directly including lay members in decision making panels.
  - Recommended
```

Reply:
@jemrobinson seeing @JimMadge 's comment, shall we remove this?

Reply:
Yes